build-support/bin/changelog.py

```diff
@@ -19,6 +19,8 @@
 from common import die
 from packaging.version import Version
 
+from pants.util.strutil import softwrap
```

Comment:
This means that you cannot run the script directly anymore. I don't think it's worth it